### DISCUSSION TITLE
Consulta em Lote NFE e NFCE

### DISCUSSION THREAD

**Discussion by maycon2-0** [TadaSoftware/PyNFe#166] [Q&A]
Seria possivel fazer a consulta das notas emitidas em lote?

**Comment by leogregianin** [comment]
> Seria possivel fazer a consulta das notas emitidas em lote?

O serviço disponibilizado para consultar todas as NF-es emitidas contra o seu CNPJ é chamado de DistribuicaoDFe. 
Não existe serviço para consulta de NFC-e.
Ele tem 3 formas de consultas descritas [neste link](https://github.com/TadaSoftware/PyNFe/wiki/Consulta-Distribui%C3%A7%C3%A3o).
O fluxo de como consultar está [neste link](https://github.com/TadaSoftware/PyNFe/wiki/Fluxo-da-Consulta-de-Distribuicao).
Tem um tópico sobre esse assunto e as mudanças constantes desse serviço [neste link](https://github.com/TadaSoftware/PyNFe/discussions/170).

**Reply by leogregianin** [reply]
@maycon2-0 para melhor organização das discussões do projeto, pode marcar essa discussão por respondida?
